### PR TITLE
Problem AbstractBootstrapInitializerTest#init solved

### LINKED ISSUE #7538: Problem with AbstractBootstrapInitializerTest
There are 2 problems in AbstractBootstrapInitializerTest.

1. Test cases are too slow, it spend 13 seconds in my laptop.

2.  When run `AbstractBootstrapInitializerTest` (run subclass `GovernanceBootstrapInitializerTest` and `StandardBootstrapInitializerTest` together), exception thrown:

```
Exception in thread "Thread-2" java.lang.ExceptionInInitializerError
	at org.apache.shardingsphere.proxy.frontend.ShardingSphereProxy.start(ShardingSphereProxy.java:65)
	at org.apache.shardingsphere.proxy.init.impl.AbstractBootstrapInitializer.init(AbstractBootstrapInitializer.java:60)
	at org.apache.shardingsphere.proxy.init.impl.AbstractBootstrapInitializerTest.triggerAbstractBootstrapInitializerInit(AbstractBootstrapInitializerTest.java:87)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.Integer
	at org.apache.shardingsphere.proxy.backend.context.BackendExecutorContext.<init>(BackendExecutorContext.java:36)
	at org.apache.shardingsphere.proxy.backend.context.BackendExecutorContext.<clinit>(BackendExecutorContext.java:33)
	at org.apache.shardingsphere.proxy.frontend.ShardingSphereProxy.start(ShardingSphereProxy.java:65)
	at org.apache.shardingsphere.proxy.init.impl.AbstractBootstrapInitializer.init(AbstractBootstrapInitializer.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mockito.internal.creation.bytebuddy.MockMethodAdvice.tryInvoke(MockMethodAdvice.java:201)
	at org.mockito.internal.creation.bytebuddy.MockMethodAdvice.access$400(MockMethodAdvice.java:29)
	at org.mockito.internal.creation.bytebuddy.MockMethodAdvice$SuperMethodCall.invoke(MockMethodAdvice.java:157)
	at org.mockito.internal.creation.bytebuddy.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:131)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:97)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:32)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:36)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:57)
	at org.mockito.internal.creation.bytebuddy.MockMethodAdvice.handle(MockMethodAdvice.java:99)
	... 3 more
```

### PR DESCRIPTION
**Pull request by sluk3r**
Fixes #7538.

Changes proposed in this pull request:
- sleep time changed to 900 ms
- for two child class, proxy port set to 2020 and 3030 respectively

### CODE REVIEW

**Review by terrymanu — CHANGES_REQUESTED** [review]
I refactor some place, but this pr revert my changes, what is the reason?

**Comment by sluk3r** [comment]
> I refactor some place, but this pr revert my changes, what is the reason?

restored

File: shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/java/org/apache/shardingsphere/proxy/init/impl/AbstractBootstrapInitializer.java [hunk 1]
```diff
@@ -55,6 +55,8 @@
 @Slf4j
 public abstract class AbstractBootstrapInitializer implements BootstrapInitializer {
     
+    private ShardingSphereProxy shardingSphereProxy = new ShardingSphereProxy();
```
Comment (terrymanu) [inline shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/java/org/apache/shardingsphere/proxy/init/impl/AbstractBootstrapInitializer.java:58]: It is better to make the field as final

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/java/org/apache/shardingsphere/proxy/init/impl/AbstractBootstrapInitializer.java (hunk 1 above)
Comment (sluk3r) [reply shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/java/org/apache/shardingsphere/proxy/init/impl/AbstractBootstrapInitializer.java]: make the field as final

**Review by terrymanu — APPROVED** [review]

**Pull request merged**